projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -203,4 +204,15 @@ public function add_options_menu( $wp_admin = false ) {
 
 		parent::add_options_menu( $wp_admin );
 	}
+
+	/**
+	 * Adds a WordPress.org theme install menu item.
+	 *
+	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
+	 */
+	public function add_theme_install_menu( $wp_admin = false ) {
+		$parent_menu_slug = $wp_admin ? 'themes.php' : 'https://wordpress.com/themes/' . $this->domain;
+
+		add_submenu_page( $parent_menu_slug, esc_attr__( 'WordPress.org Themes', 'jetpack' ), __( 'WordPress.org Themes', 'jetpack' ), 'switch_themes', 'theme-install.php', null, 1 );
```

Comment:
Any specific reason we can't go with just "Themes"? I wonder if "WordPress.org" here could be more confusing than anything else.

Reply:
We already have a "Themes" menu `https://wordpress.com/themes/[DOMAIN]`.

Reply:
I see your point about the "confusion" though!

Reply:
See https://github.com/Automattic/jetpack/pull/18780#pullrequestreview-588420913